src/platform/plugins/shared/controls/public/controls/types.ts

```diff
@@ -36,7 +36,6 @@ export type DefaultControlApi = PublishesDataLoading &
   PublishesBlockingError &
   PublishesUnsavedChanges &
   Partial<PublishesTitle & PublishesDisabledActionIds & HasCustomPrepend> &
-  CanClearSelections &
```

Comment:
Instead of removing `CanClearSelections` from `DefaultControlApi`, how about just `Partial<CanClearSelections>`

Reply:
Sure!